elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/Column.java

```diff
@@ -79,11 +81,15 @@ protected Column(Table table, String fieldName, EntityDictionary dictionary) {
             this.category = meta.category();
             this.values = new HashSet<>(Arrays.asList(meta.values()));
             this.tags = new HashSet<>(Arrays.asList(meta.tags()));
+            this.tableSource = (meta.tableSource().trim().isEmpty()) ? NONE : meta.tableSource();
```

Comment:
I believe the UI will interpret this as an actual ID and look it up in the metadata API.  Better to just let it be null.